opnode/flags/flags.go

```diff
@@ -63,13 +63,6 @@ var (
 		EnvVar: prefixEnvVar("SEQUENCING_ENABLED"),
 	}
 
-	// TODO: move batch submitter to stand-alone process
```

Comment:
nit: Unrelated to p2p work.

Reply:
It's an outdated comment that needed to be removed, and I wanted to avoid confusion with the sequencer p2p key by cleaning up any unused references to the batch submitter key.